src/client/containers/SlackAuthPage/SlackAuthPage.component.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+
+const axios = require('axios');
+
+export default function SlackAuthPage() {
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
```

Comment:
Also we could you the react-router-dom to get the parameters instead of accessing it through the `window` object.

Reply:
so do i need to change the code?

Reply:
I suggest you try it out too - when you use `react-router-dom` in applications it is assumed that you will use it consistently.
To get the current search, use the `useLocation` hook.